integration_tests/utils.js

```diff
@@ -162,7 +162,7 @@ const cleanupStackTrace = (output: string) => {
     .replace(/\n.*at.*assert\.js.*$/gm, '')
     .replace(/\n.*at.*node\.js.*$/gm, '')
     .replace(/\n.*at.*next_tick\.js.*$/gm, '')
-    .replace(/\n.*at (new)? Promise \(<anonymous>\).*$/gm, '')
+    .replace(/\n.*at (new )?Promise \(<anonymous>\).*$/gm, '')
```

Comment:
Should we have unit tests for out test utils? 😀

Reply:
I won't be the one writing them 😄

Reply:
FWIW I think this should be stripped in here:https://github.com/facebook/jest/blob/8a427ebba88c6ab15da7c987716216728f5ea093/packages/jest-message-util/src/index.js#L150-L166

It being in the stack is not helpful at all (along with `at anonymous` and `process._tickCallback` (the latter should be stripped by `removeInternalStackEntries` as well, shouldn't it?))

![image](https://user-images.githubusercontent.com/1404810/31306129-b0caefca-ab49-11e7-8db7-2ecce7512005.png)

Only the first line in the stack is useful

Reply:
I'm a fan of removing `at process._tickCallback` and `at <anonymous>` without any further info (like line number) from the stack trace 👍.

Reply:
If we do that, I think you can remove the line with a diff in this PR entirely

Reply:
I'm happy to modify this PR or send a followup when we have the discussed behavior merged.

Reply:
Yeah, this should probably be merged in the meantime